build.gradle.kts

```diff
@@ -26,6 +27,9 @@ dependencies {
 
     compile("org.ahocorasick", "ahocorasick", ahoCorasickVersion)
     compile("com.google.guava", "guava", guavaVersion)
+    compile("edu.stanford.nlp", "stanford-corenlp", stanfordNlpVersion)
+    compile("edu.stanford.nlp:stanford-corenlp:$stanfordNlpVersion:models")
+    compile("edu.stanford.nlp:stanford-corenlp:$stanfordNlpVersion:models-french")
```

Comment:
Why are we depending on a french model?

Reply:
Oops. This was a hangover from when I was trying to get French lemmatisation working. Will remove